internal/uvm/constants.go

```diff
@@ -16,6 +16,19 @@ const (
 	// DefaultVPMemSizeBytes is the default size of a VPMem device if the create request
 	// doesn't specify.
 	DefaultVPMemSizeBytes = 4 * 1024 * 1024 * 1024 // 4GB
+
+	// LCOWMountPathPrefix is the path format in the LCOW UVM where non global mounts, such
+	// as Plan9 mounts are added
+	LCOWMountPathPrefix = "/mounts/m%d"
```

Comment:
I think largely this type of constant should be defined in `hcsoci`. The role of `hcsoci` seems to be to map from container concepts into the specifics of how things need to work with our UVM system, so I think these constants make more sense there.

However, it looks like there are some issues with doing that for all of these right now, as that would cause circular dependencies between `hcsoci` and `layers`. My impression is we will need to sort out the knot between those two packages into a cleaner interface, but that can be done at a later date. What we have here looks like a good start.

Reply:
I agree, though I think we'll need to sort through better how we want to do that and how we should be writing files/packages moving forward to avoid this weird tangle.